BrowserStack.java

```diff
@@ -37,32 +38,113 @@ public static void myassert(boolean  x) {
     //push() - pushes the element in this case its a url (String)
     private  void push(String url){
         //WRITE YOUR CODE HERE
+    	
+    	top ++;
+    	myStack.add(top, url);
+    	    	
+    		
     }
     //pop()- pops the element from the stack
     private String pop(){
         //WRITE YOUR CODE HERE
-
+    	if (!isEmpty()){
+		 String popUrl = myStack.remove(top);
```

Comment:
The expectation is to write the code to remove the element from array. I understand since you have used Arraylist you are forced to use ArrayList methods. Would you be able to write this using array instead of ArrayList